docs/lifecycle/lifecycle.rst

```diff
@@ -117,36 +129,129 @@ Board LFN Entry Review
 ++++++++++++++++++++++
 
 It is up to the Board to define its own criteria and process of the Board's LFN
-Entry Review. The TAC recommends the Board make its LFN Entry Review criteria
-and process public and accept design input from the public.
+Entry Review. Such review may include legal, trademark, and license reviews. The 
+TAC recommends the Board make its LFN Entry Review criteria and process public 
+and accept design input from the public.  
+
+Budget Guidance: The TAC recommends that the governing board allocate no funding 
+to Sandbox projects and that those projects cannot use cross-project funding
+buckets.
+
+Quarterly Health Review
+***********************
+
+The TAC should review all projects in the Sandbox state on a quarterly basis. 
+The goal of the quarterly health review is to assist the projects in their 
+participation in LFN and ensure the project's community remains healthy.
+
+Incubation Review
+*****************
+
+The Board and the TAC both review proposals for projects to advance to the 
+Incubation state.
+
+TAC Incubation Review
++++++++++++++++++++++
+
+TAC Incubation Reviews should happen before, and provide input into, Board 
+Incubation Reviews.
+
+Required Information for TAC Incubation Review
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An up-to-date instantiation of the Project Data Template is required for an 
+Incubation Review.
+
+Criteria for Incubation Review
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Mandatory criteria for successful completion of the Incubation review is
+the documented demonstrable progress since induction toward open source best practices. This would include but is not limited to contributor diversity and 
+open governance.
+
+As an additional (non-mandatory) guideline, the project should show demonstrable
+progress in marketplace adoption.  This may be measured by instances of the
+project in production or the project being embedded in other open source 
+projects.
+
+As part of the Incubation Review, the TAC should identify how the project fits
+with other LFN projects, including any overlap or harmonization potential.
+
+Outcome for Incubation Admission Review
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As an outcome of the TAC's Incubation Review, the TAC will provide the following
+feedback to the LFN Governing Board for use as input to the LFN Board's 
+Incubation Review:
 
+* Summary of findings
+* Recommendation to accept the project into the Incubation stage or not.
+
+Board Incubation Review
++++++++++++++++++++++++
+
+It is up to the Board to define its own criteria and process of the Board's 
+Incubation Review. The TAC recommends the Board make its Incubation Review 
+criteria and process public and accept design input from the public.  
```

Comment:
See comment above about recommending that Incubation project funding not come at the expense of TAC projects.